src/views/admin/AdminPerformanceView.vue

```
@@ -0,0 +1,1174 @@
+    memberCount: Number(node.memberCount || 0),
+    children,
+  }
+}
+
+const sortNodes = (nodes = []) => {
+  nodes.sort((a, b) => {
+    if (a.sortOrder !== b.sortOrder) return a.sortOrder - b.sortOrder
+    if (a.level !== b.level) return a.level - b.level
+    return String(a.name || '').localeCompare(String(b.name || ''), 'ko')
+  })
+  nodes.forEach((node) => sortNodes(node.children))
```

Biome (2.4.7)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)